pkg/sdkserver/localsdk.go

```diff
@@ -69,6 +70,12 @@ func defaultGs() *sdk.GameServer {
 			State:   "Ready",
 			Address: "127.0.0.1",
 			Ports:   []*sdk.GameServer_Status_Port{{Name: "default", Port: 7777}},
+			Counters: map[string]*sdk.GameServer_Status_CounterStatus{
```

Comment:
So this should only be added when `runtime.FeatureEnabled(runtime.FeatureCountsAndLists)`.

I expect you'll need to set the `&sdk.GameServer` to a temporary variable before returning.

Reply:
I have conditionally added the Counters and Lists when the feature is enabled and the value of sdk.GameServer is stored in a temporary variable.